dev.yml

```diff
@@ -1,7 +1,7 @@
 name: active-fulfillment
 
 up:
-  - ruby: 2.3.1
+  - ruby: 2.6.5
```

Comment:
couldn't run tests locally using the old version of ruby. This shouldn't impact anything else